config/mongo.go

```diff
@@ -57,3 +63,43 @@ func LoadMongoDBFromEnv() *MongoDB {
 	}
 	return &mongo
 }
+
+func (m *MongoDB) mustMode(s *mgo.Session) {
+	if m.Mode == "" {
+		return
+	}
+
+	var mode mgo.Mode
+
+	switch m.Mode {
```

Comment:
Mind switching this to `strings.ToLower` and changing the strings to match? It prevents us from being case sensitive in the configs.